src/style.css

```diff
@@ -1,3 +1,77 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap';
+
 body {
   margin: 0;
+  background-color: #e1e3e1;
+  font-family: Roboto, sans-serif, system-ui;
```

Comment:
Consider adding a fallback font-family in case 'Roboto' isn't available. For example, you could use a standard font like Arial as a fallback: `font-family: 'Roboto', Arial, sans-serif;`.